underscore.js

```diff
@@ -1105,8 +1104,22 @@
     return _.pick(obj, iteratee, context);
   });
 
-  // Fill in a given object with default properties.
-  _.defaults = createAssigner(_.allKeys, true);
+  // Fill in an object with default properties.
+  _.defaults = function() {
+    var dest = {};
+    for (var index = arguments.length - 1; index >= 0; index--) {
```

Comment:
Right, so it's not really `_.defaults` and not really `_.extendRight`. I think the way I went is the lesser of worse ways to go in this situation.